### PR TITLE
Feature display movies

### PR DESCRIPTION
**Pull request by saraho1123**
## Description
We completed iteration 1. Yay! We added several components in order to display the movie data we were given. Components:
- App
- Header
- MovieContainer
- MovieCard
- SingleMovie (we are saving this one for future iterations!)

## Motivation and Context

It solves the problem of getting the movies to display! ;) And completing iteration 1!

## How Has This Been Tested?
This is the beginning of our code and we have not yet started testing. However, it can be tested by running npm start from the terminal (after cloning and running npm install). When npm start is run, the page should display with the header and all the lovely movie cards!

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation. n/a
- [ ] I have updated the documentation accordingly. n/a
- [ ] I have added tests to cover my changes. n/a (yet)
- [x] All new and existing tests passed.
 
## Tickets: 

#3 #4 #5 #6 #7 #8

### CODE REVIEW

File: src/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+header {
+  text-align: center;
```
Comment (saraho1123) [inline src/Header/Header.css:2]: I like our page! We might want to do a little refactoring of styling down the road.  Also, do we want to consider a different font?

File: src/MovieCard/MovieCard.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import './MovieCard.css';
+
+const MovieCard = ( {image, title, rating}) => {
+  return (
+    <section className="movie-card">
+      <div className="image-holder">
+        <img className="card-image" src={image}/>
+      </div>
+      <h3 className="card-title">Movie Title: {title}</h3>
+      <h3 className="rating">Movie Rating: {Math.round( rating * 10 ) / 10}</h3>
```
Comment (chadrick-d-dev) [inline src/MovieCard/MovieCard.js:11]: I'm glad we rounded up the ratings!

File: src/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  background: green;
+}
+
+.heading {
+  margin: 0;
+  padding: 1em;
+  color: rgb(74, 35, 110);
+}
```
Comment (saraho1123) [inline src/Header/Header.css:10]: Is it important to have an empty line at the end of js files?

File: src/MovieCard/MovieCard.js (hunk 2 above)
Comment (saraho1123) [inline src/MovieCard/MovieCard.js:11]: I love that we can put a method like this in the JSX! Learning so much! This is fun!

**Review by chadrick-d-dev — APPROVED** [review]
Home run! I think we did an amazing job together, today!

**Review by saraho1123 — COMMENTED** [review]
We are on a role now! Let's keep it up!

File: src/Header/Header.css (hunk 3 above)
Comment (chadrick-d-dev) [reply src/Header/Header.css:10]: I don't think it is.

File: src/Header/Header.css (hunk 1 above)
Comment (chadrick-d-dev) [reply src/Header/Header.css:2]: I agree. Refactoring and styling will be good.

**Pull request merged**